egs/lre07/v1/lid/init_full_ubm_from_dnn.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# Copyright 2015   David Snyder
+#           2015   Johns Hopkins University (Author: Daniel Garcia-Romero)
+#           2015   Johns Hopkins University (Author: Daniel Povey)
+# Apache 2.0
+
+# This script derives a full-covariance UBM from DNN posteriors and
+# language recognition features.
+
+# Begin configuration section.
+nj=40
+cmd="run.pl"
+stage=-2
+
+# End configuration section.
+
+echo "$0 $@"  # Print the command line for logging
+
+if [ -f path.sh ]; then . ./path.sh; fi
+. parse_options.sh || exit 1;
+
+if [ $# != 4 ]; then
+  echo "Usage: steps/init_full_ubm_from_dnn.sh <data-speaker-id> <data-dnn> <dnn-model> <new-ubm-dir>"
+  echo "Initializes a full-covariance UBM from DNN posteriors and speaker recognition features."
+  echo " e.g.: steps/init_full_ubm_from_dnn.sh data/train data/train_dnn exp/dnn/final.mdl exp/full_ubm"
+  echo "main options (for others, see top of script file)"
+  echo "  --config <config-file>                           # config containing options"
+  echo "  --cmd (utils/run.pl|utils/queue.pl <queue opts>) # how to run jobs."
+  echo "  --nj <n|16>                                      # number of parallel training jobs"
+  echo "  --delta-window <n|3>                             # delta window size"
+  echo "  --delta-order <n|2>                              # delta order"
```

Comment:
I don't think delta-window or delta-order are used in this script. Please remove.